### ISSUE TITLE
Configurable ServiceName

### ISSUE THREAD

**Issue by bforbis** [Kong/kong-plugin-zipkin#119]
Currently the serviceName is hard-coded to "kong". Could this be made a plugin configuration?

**Comment by kikito** [comment]
Yes, but it would be easier and simpler to use a custom tag instead, with the `static-tags` config option. In json it would look like this:

```
"config": {
  ...
  "static_tags": [ { "name": "my-service-name", "value": "my-value" } ]
}
```
Then you could use that tag to filter/search in Zipkin

**Comment by bforbis** [comment]
I get that it is possible to do static_tags now, however it doesn't really provide me with what I want. When viewing a trace, I want to be able to control the name of the service as it is displayed in zipkin/jaeger. Tags are only used for searching, not for displaying the actual spans in the trace view. It would also be strange for kong to be the only service in my set of services where I would want to search by the name tag rather than by the service name.

**Comment by kikito** [comment]
I have marked this as an enhancement for future versions of the plugin. I have very little bandwidth to dedicate to new features and bugfixes, so it might take me a while to get to implementing that. The best way to improve the odds of including this feature on future versions of the plugin would be sending a PR.

**Comment by bforbis** [comment]
@kikito I have put together a PR for this, would you mind reviewing it?
